avm/ptn/lz/sub-vending/src/modules/Microsoft.Authorization/locks/.parameters/rg.parameters.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Are these modules needed or can they be replaced with AVM modules?

If they are needed, I'd recommend to strip them down to only what's necessariy - that is the `main.bicep`, `readme.md` and `version.json` files

Reply:
We had other CARML modules that we converted to AVM so those are the minimum we need to avoid having breaking changes. I just updated the PR with removing unnecessary files from them 👍

Reply:
Resolved

Reply:
Resolved